main.html

```diff
@@ -251,7 +268,11 @@ <h2>About Me</h2>
           <article class="about-me__card__container languages">
             <div class="about-me__languages">
               <div class="about-me__icon__holder">
-                <img src="./assets/worldicon.png" alt="World vectorized icon" srcset="" />
+                <img 
+                src="./assets/worldicon.png" 
+                alt="World icon" 
+                srcset="" 
+                />
```

Comment:
You did a good job with the about section, but i'd like to make the following changes to help you have a better user experience and design similar to the Figma design :

- [ ] Please add the background image on the extreme right of the about section to help your design look better 😄 
- [ ] Also, the content in the about section ` Languages, Framework and skills` ahould be centered and all on teh same line just as it is in the Figma template given 👍 

Figma design :

![image](https://github.com/rudicarrilloypr/portfolio/assets/20537712/4b85e90b-de97-4cd6-b9cb-755742d9b2df)

Your design :

![image](https://github.com/rudicarrilloypr/portfolio/assets/20537712/76752a11-477f-4499-a6e7-919f6d2b1c35)

![image](https://github.com/rudicarrilloypr/portfolio/assets/20537712/a18fb163-c8b1-43c2-bd99-388766cbc7e8)